unity-environment/Assets/ML-Agents/Scripts/CoreBrainPlayer.cs

```diff
@@ -77,7 +78,7 @@ public void DecideAction(Dictionary<Agent, AgentInfo> agentInfo)
         {
             foreach (Agent agent in agentInfo.Keys)
             {
-                var action = new float[brain.brainParameters.vectorActionSize];
+                float[] action = Enumerable.Repeat(0.5f, brain.brainParameters.vectorActionSize).ToArray();
```

Comment:
I think there is a strong assumption here that the developer will ALWAYS have continuous actions rescaled with opposite min and max values. I do not think we should make this assumption. Instead, the NN should output between -1 and 1 and the rescale should be a multiplication instead of the `ScaleContinuousAction` method.